app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -171,7 +171,6 @@ def save_key_pairs_inventory(ems, hashes, target = nil)
   def save_cloud_volumes_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    return unless ems.cloud_volumes
```

Comment:
@roliveri Good question. Actually these conditionals were added by me, just for spec tests. But now I think we may really need them. I'll do some tests on this. Thanks,